### ISSUE TITLE
Test postgres base backup using wal-g

### ISSUE THREAD

**Issue by Firefishy** [openstreetmap/operations#883]
wal-g has the ability to create base backups and publish these to S3.

Previously the version of postgres we were using was too old for this functionality.

Test if wal-g can create base backups.

**Comment by tomhughes** [comment]
After upgrading wal-g to 2.0.1 I was able to do a base backup using direct filesystem accress, which completed in about five seconds because it only backed up the base directory with the main tablespace.

It's supposed to be able to do other tablespaces and I can't see anything obvious in the code that is wrong or that would stop it working unless it is something that has been fixed since the 2.0.1 release last August.

I could try a replication connection based backup but https://github.com/wal-g/wal-g/pull/1396 suggests that is currently broken for Postgres 15.

**Comment by Firefishy** [comment]
New PR for Postgres 15 support seems to be https://github.com/wal-g/wal-g/pull/1595

**Comment by Firefishy** [comment]
[Looks like a PG15 base backup fix finally was merged into wal-g](https://github.com/wal-g/wal-g/pull/2262). It doesn't look like it has landed in a release yet.

**Comment by tomhughes** [comment]
Yes but we're not going to test it with `wal-g` at this point when we're looking to migrate off that, we'll test it with whatever we replace it with.
